storage_service/administration/forms.py

```diff
@@ -222,8 +222,17 @@ class UserChangeForm(auth.forms.UserChangeForm):
     """ Modifys an existing user.  Inherits from django's UserChangeForm. """
 
     def __init__(self, *args, **kwargs):
+        self.current_user = kwargs.pop("current_user", None).username
         super(UserChangeForm, self).__init__(*args, **kwargs)
+        currently_editing = getattr(kwargs["instance"], "username")
+        superusers = auth.get_user_model().objects.filter(is_superuser=True)
         self.fields["is_superuser"].label = _("Administrator?")
+        if len(superusers) >= 1:
```

Comment:
Minor note because this is a super common Django ORM gotcha/thing: `len(superusers)` selects all user data for super users, builds a list, and checks the list length, while `superusers.count()` will do a `SELECT COUNT...`.  Not really an issue here, moreso if the number of items in the list is likely to be large.